### ISSUE TITLE
Why have we forked pefile?

### ISSUE THREAD

**Issue by botherder** [viper-framework/viper#664]
I'm just noticing we're not using the actual pefile repository and we just forked it in here. Why?

**Comment by Rafiot** [comment]
Because pefile isn't properly working with python3, the python3 fixes were taking months to be merged, if they were (there is a PR iirc), and the test cases are private, making it almost impossible to validate the changes.

it might be better now, but I'm planning to write a lief (https://github.com/lief-project/LIEF) module instead: the library is maintained, super fast, does the same things as pefile (and more), supports PE, ELF, and MACH-O, and will get android support in the next release.

**Comment by botherder** [comment]
Have you made any progress on this?
If not, I can take a crack at it.

**Comment by Rafiot** [comment]
No, I haven't :/ So yeah, if you can take a crack at it, that would be great.
